package.json

```diff
@@ -69,7 +70,7 @@
     "eslint-mdx": "^2.1.0",
     "eslint-plugin-eslint-plugin": "^5.1.0",
     "eslint-plugin-mdx": "^2.1.0",
-    "eslint-plugin-n": "^16.0.1",
+    "eslint-plugin-n": "^16.3.1",
```

Comment:
The CI is broken after upgrading.

Reply:
[eslint-plugin-n v16 dropped support for node <16](https://github.com/eslint-community/eslint-plugin-n/releases/tag/16.0.0), and 16.1.0 introduced some syntax - `??=` that is only available in v15 and up. Annoyingly v16.1.0 is also the version that introduced the `flat/recommended` config that we need so we can't pin to a version that's got just the thing we need.